build/Targets/Settings.props

```diff
@@ -187,21 +177,18 @@
     </When>
   </Choose>
 
-  <!-- Visual Studio specific properties -->
+  <!-- Determine the appropriate build tools to us.  When doing an official build we need to use the RC2 SDK
+       in order to produce the correct V3 VSIX.  Otherwise default to the current Dev15 build the code 
+       is sync'd with -->
   <Choose>
-    <When Condition="'$(VisualStudioVersion)' == '15.0' AND '$(IsVSBeforeDev15Preview4)' == 'true'">
-      <PropertyGroup>
-        <VisualStudioBuildToolsVersion>15.0.25201-dev15preview2</VisualStudioBuildToolsVersion>
-      </PropertyGroup>
-    </When>
-    <When Condition="'$(VisualStudioVersion)' == '15.0' AND '$(IsDev15RC)' == 'true'">
+    <When Condition="'$(OfficialBuild)' == 'true'">
       <PropertyGroup>
         <VisualStudioBuildToolsVersion>15.0.25825-RC</VisualStudioBuildToolsVersion>
       </PropertyGroup>
     </When>
-    <When Condition="'$(VisualStudioVersion)' == '15.0'">
+    <When Condition="'$(IsDev15Vsix)' == 'true'">
```

Comment:
@Pilchie and if we want to force every extension to always build with our current Dev15 SDK version then we just delete this condition and make this the otherwise block.